src/Robo/Hooks/InteractHook.php

```diff
@@ -98,4 +99,31 @@ public function interactExecuteUpdates(
     }
   }
 
+  /**
+   * Confirms active config matches config in sync directory.
+   *
+   * @hook interact @interactConfigIdentical
+   */
+  public function interactConfigIdentical(
+    InputInterface $input,
+    OutputInterface $output,
+    AnnotationData $annotationData
+  ) {
+    if ($this->getConfigValue('cm.strategy') == 'config-split') {
+      $drupal_installed = $this->getInspector()->isDrupalInstalled();
+      $config_identical = $this->getInspector()->isActiveConfigIdentical();
+      if (!$config_identical) {
+        $this->logger->warning("The site's active config does not match the config in the sync directory.");
```

Comment:
I'd use the same language as drush `The active configuration is not identical to the configuration in the export directory`.